app/domain/authentication/authn_jwt/fetch_mandatory_claims_to_validate.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Authentication
+  module AuthnJwt
+    OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
+
+    # FetchMandatoryClaimsToValidate command class is responsible to return a list of mandatory JWT standard claims to
+    # validate.
+    FetchMandatoryClaimsToValidate ||= CommandClass.new(
+      dependencies: {
+        fetch_issuer_value: ::Authentication::AuthnJwt::FetchIssuerValue.new,
+        logger: Rails.logger
+      },
+      inputs: %i[authentication_parameters]
+    ) do
+
+      def call
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingJwtMandatoryClaimsToValidate.new)
+        fetch_mandatory_claims_to_validate
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchedJwtMandatoryClaimsToValidate.new)
+
+        mandatory_claims_to_validate
+      end
+
+      private
+
+      # fetch_mandatory_claims_to_validate function is responsible to return a list of mandatory JWT standard claims to
+      # validate, according to the following logic:
+      # For each optional claim (iss, exp, nbf, iat) that exists in the token - add to mandatory list
+      # Note: mandatory list also contains the value to validate if necessary (for example iss: cyberark.com)
```

Comment:
can you move this comment to the top of the class? it seems that this function does all the logic so its logic is the same as the class.